validator/client/validator_attest.go

```diff
@@ -172,7 +146,14 @@ func (v *validator) SubmitAttestation(ctx context.Context, slot uint64, pubKey [
 		}
 		return
 	}
-
+	indexedAtt.Signature = sig
+	if err := v.postSignatureUpdate(ctx, indexedAtt, pubKey); err != nil {
+		log.WithFields(logrus.Fields{
+			"sourceEpoch": indexedAtt.Data.Source.Epoch,
+			"targetEpoch": indexedAtt.Data.Target.Epoch,
+		}).Fatal("made a slashable attestation, found by external slasher service")
```

Comment:
Why fatal here? The slasher should block the slashable attestation instead of  just`fatal` imo

Reply:
It should block in the pre attestation
It tries to update the db at this stage. If a slashing is found it means that a slashing offence has been already sent to network. I even thought of panic

Reply:
Blocking by fatal? Maybe this should be an error log or warn, can it be blocked a way thats less scary for the user?

Reply:
maybe you got me wrong. there is a validation stage before signing the attestation. 
if a slashble is detected in the post signature stage it means that slashble attestation has been transmitted to the network. the user should be scared and try to find whats going on. maybe he got two validators running and stopping one of them will prevent it from recurring